packages/commerce-sdk-react/README.md

```diff
@@ -26,17 +26,23 @@ const AppConfig = ({children}) => {
     )
 } 
 
+// Custom retry logic
+// Ref: https://tanstack.com/query/v4/docs/guides/query-retries
+const shouldRetry = (failureCount, error) => {
+    // For example: do not retry on server side, but keep retrying until max of 3 failures on the client-side
+    return isServer ? false : failureCount < 3
```

Comment:
nit: The counter will be initialized to `failureCount = 1` so if we want to retry 3 times we'll need to update the condition.
```suggestion
    return isServer ? false : failureCount <= 3
```